src/examples/upload_package_using_poetry.yml

```
@@ -0,0 +1,27 @@
+description: >
+  Upload a package from Cloudsmith using poetry.
+  For uploads, the source URL in pyproject.toml must be set to https://python.cloudsmith.io/financial-times/your-repository-id/
+  and the corresponding parameter cloudsmith-source-url is an arbitrary label for the Cloudsmith Source URL.
+usage:
+  version: '2.1'
+  orbs:
+    cloudsmith-python: ft-circleci-orbs/cloudsmith-python@1.0
```

```suggestion
    cloudsmith-python: ft-circleci-orbs/cloudsmith-python@1.1
```
We should update this and other examples as this will change to 1.1 when we release the orb.